codes/.env

```diff
@@ -0,0 +1 @@
+LANGS_SDK="python cpp java csharp"
```

Comment:
The languages to be installed are selected using `LANGS` arg in Dockerfile. Is this duplicated?

Reply:
Actually not, the LANGS in the dockerfile entirely depend on the variables in the docker-compose file. I think we need a default value here, but the dockerfile still needs to update.

Reply:
OK, I removed the default values of `LANGS` in Dockerfile to make it fully controlled by `docker-compose`.